### ISSUE TITLE
WCAG 2.2: Images ‖ Target Size (Minimum)

### ISSUE THREAD

**Issue by querkmachine** [alphagov/govuk-design-system#2881]
## What

Update images guidance to specify a minimum set of dimensions that linked images should have.

### Resources

- [Images styles page](https://design-system.service.gov.uk/styles/images/)
- [WCAG 2.2 criterion for 'Target Size (Minimum)'](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html)

## Why

The new WCAG 2.2 Level AA Criterion 'Target Size (Minimum)' says: 

> The size of the [target](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html#dfn-target) for [pointer inputs](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html#dfn-pointer-input) is at least 24 by 24 [CSS pixels](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html#dfn-css-pixel)

Although it feels unlikely for a service team to use images smaller than this in public-facing services, case working tools may have smaller controls (for example, a toolbar on a WYSIWYG editor). We may want to include specific guidance about minimum sizes for linked or interactive images and iconography.

## Who needs to work on this
- Content designer or technical writer

## Who needs to review this
- Accessibility specialist
- Content designer or technical writer

## Done when

- [ ] Guidance has been updated

**Comment by calvin-lau-sig7** [comment]
Closing issue – content has been drafted, reviewed and added to pull request #3090 as part of https://github.com/alphagov/govuk-design-system/issues/3276.
